client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.tsx

```diff
@@ -298,7 +298,7 @@ export function WPNonProductLineItem( {
 		<div
 			className={ joinClasses( [ className, 'checkout-line-item' ] ) }
 			data-e2e-product-slug={ lineItem.id }
-			data-product-type={ lineItem.id }
+			data-product-type={ lineItem.type }
```

Comment:
@sirbrillig I've been trying to fix the end to end tests for this PR and it seems like the failures related to removing a coupon were caused because of [this change to an attribute value](https://github.com/Automattic/wp-calypso/pull/49581/files#diff-7dcbfb6b6cb0263c9baa37599f7513ff164d19687374fd507f2c34da36ce1a08L97). Could you check it for me, please?

Reply:
🤔  Yes, that seems like a regression on my part. `data-product-type` used to be `type` (which is typically either `plan` or the product slug) but here I accidentally changed it to `id`. Thanks for figuring that out and I apologize for the error!

Was there anything else that you were still wondering about?

Reply:
Thanks for that @sirbrillig. I just wanted to check my assumptions as I'm not familiar with that part of the codebase, and I definitely didn't want to break the checkout process!

I think that's all you can help me with for now. I've got some theme related tests failing too, which I'm trying to puzzle out, but they're not related to any of your changes.